index.html

```diff
@@ -231,23 +231,25 @@ <h3>Latest news</h3>
         -->
         </div>
         <div class="js-home-carousel home-carousel">
+
           <!-- News box Start -->
-          <div class="news-box">
+	  <div class="news-box">
             <div class="news-box-title">
-              <span>31/01/2024</span>
+              <span>16/02/2024</span>
             </div>
             <div class="news-box-content">
-              <p>Uyuni 2024.01 is now available with openEuler 22.03 support, reboot required indicator for non-SUSE distributions... For uyuni-tools in deb-based systems, please check the release announcement. Check the <a href='pages/stable-version.html#releasenotes'>release notes</a> for the full details!</p>
+              <p>Uyuni 2024.02 brings important CVEs fixes. Please, read carefully the <a href='pages/stable-version.html#releasenotes'>release notes!</a> The Uyuni server container image is also available for aarch64 (ARM64) and mgradm provides new start/stop/restart commands.</p>
```

Comment:
Question of tests, but I'd mention the reboot flag instead of start/stop/restart for mgradm :-)

Reply:
That was already included in 2024.01

Reply:
Ah, right. I probably got confused by SUSE Manager release notes.